src/transport_ofi.c

```diff
@@ -86,6 +87,16 @@ static char                     myephostname[EPHOSTNAMELEN];
 #endif
 #ifdef ENABLE_THREADS
 shmem_internal_mutex_t          shmem_transport_ofi_lock;
+/* Need a syscall to gettid() because glibc doesn't provide a wrapper
+ * (see gettid manpage in the NOTES section): */
```

Comment:
Should we add a configury check to see if ```gettid()``` already exists?  Otherwise, this will cause potential errors since ```sys/types.h``` is included above, which pulls in ```gettid()``` but it's redefined below.

Reply:
Would it help to _not_ redefine `gettid()`, and name this something like `shmem_transport_ofi_gettid()`?

Otherwise, yeah, a configury check would be fine.

Reply:
You can take a look at how this is handled for CMA.  It's pretty much the same situation.